src/common/Assert.h

```diff
@@ -66,9 +66,11 @@ void HandleAssertionFailure(const char* file, const char* function, int line, co
 #endif
 
 #define NXT_ASSERT(condition) NXT_ASSERT_CALLSITE_HELPER(__FILE__, __func__, __LINE__, condition)
+#define NXT_UNREACHABLE() NXT_ASSERT(false && "Unreachable code hit"); NXT_BUILTIN_UNREACHABLE()
```

Comment:
Semicolon in a macro is a bit scary. Is it possible to have
```
#define NXT_UNREACHABLE() do { NXT_ASSERT(false && "Unreachable code hit"); NXT_BUILTIN_UNREACHABLE(); } while(0)
```
or does that break `__builtin_unreachable`?